### DISCUSSION TITLE
Do not auto open next email

### DISCUSSION THREAD

**Discussion by akosnikhazy** [tutao/tutanota#4662] [Ideas]
**The idea:** A setting to turn off auto-open next email would be nice on desktop version. After deleting or archiving an email it should show the default empty space instead of jumping to the next email opening it. I see no setting for turn this behavior off.

**There are two reasons:**
**The user experience of user choice:** There are times when users want to keep emails unread for later consumption. By auto-opening them, the user risks forgetting, marking it unread again, or missing important messages. It is important for a lot of workflows to keep emails untouched until the user opens it by choice, not automatically. I see there are times when the present behaviour saves time, but I find it **bad design that a decision a user makes on one email (deleting it) affects another completely different email (opens it).**

**Personal one:** I know I am not alone using emails sent to myself as notes. I keep them unread while they are relevant. I did this on gmail. Now that I am on tutanota, it became annoying that when I delete an email it jumps to the next one and opens it. This way, after the last "not a note to myself" email it opens one of my "note emails" and I have to mark it as unread again. So, I kept a buffer email at the top, just to avoid this happening. Emails as notes are the best way I found as I do not have to rely on another app to make notes, also I do not have to build in another app to my workflow.

**Comment by akosnikhazy** [comment]
Now ff I use the checkbox before I delete an email, it does not open the next. Almost there!

**Comment by yellowtrolley** [comment]
Very annoying indeed. I leave unread messages as reminders and that is no use if they are marked as read.
Considering that Search feature is useless, if you want to find your stuff, you need to keep moving emails to other folders or deleting them if not important. Any of those actions will cause that next message will be marked as read.
Why? There is no such behavior in any of the other email clients that I use.
